english/ironman/ironman.md

```diff
@@ -1 +1,8 @@
-Wrong Choice.
\ No newline at end of file
+Wrong Choice.
+
+I'll give you one chance to redeem yourself, though.
+Who should win in a matchup?
+
+[Batman](english/batman/batman.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.